### PR TITLE
readme bugfix, updates, and ddm_test_validate.py fixes

### PR DESCRIPTION
**Pull request by journeyman-msft**
# Pull Request Template

## Description

README bugfixes and improvements since I was a new user. Great work guys. I added instructions for how to use ddm_test_validate.py and fixed issues I ran into. Currently it generates 2 csv's (but future functionality can include plots later-- i did this on another one my branches and found it to be really helpful)

Fixes # (issue)
image
image
- sim_orig() in ddm_test_validate.py

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Reviewer Tasks

List here all the tasks the reviewer needs to review:

- [ ] Test `examples/`

**Test Configuration**:
* OS:
* Python Version:
* Bonsai SDK Version:

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by sdk-bonsai** [comment]
We already discussed on Friday, and I was able to run your branch after instantiation. So, it runs! (Maybe someone with a mac can also test.) My original branch also used to work, but it was decided to remove simulator from `main` repository, and only comments were kept. So, this was a much needed testing! Thanks! Should have add pytests for ddm_test_validate also, as some of the updates came later and broke the previous version. You are the first person to use it after a while. I am fine with the PR. Let one of the other folks test on mac once, and then merge. For plotting purposes, there is also a notebook on visualization that  Juan and Mikayel have done some work. That is one place where plotting could be integrated.

File: README.md [hunk 1]
```diff
@@ -46,14 +46,16 @@ The scripts in this package leverage the configuration files saved in the [`conf
 
 The library comes with a default configuration set in [`conf/config.yaml`](conf/config.yaml).
 
+> NOTE: Be sure to add your actions to the `augmented_cols` instead of `inputs` in the data config file located at `conf/data`.
```
Comment (akzaidi) [inline README.md:49]: Is there a reason why we want to make this requirement? Either shoudl work. Initiailly, the design was `augmented_cols` would be for exogeneous variables, like configs only.

File: README.md [hunk 2]
```diff
@@ -90,35 +92,80 @@ The sweeping function uses [`tune-sklearn`](https://github.com/ray-project/tune-
...
 ```
 
-## Build Simulator Package
+> NOTE: The optional flags should NOT have .yml; it should just be the name of the config file
+
+If you're having trouble running locally, chances are you need to set up your workspace and access key configs. You can do this by using environment variables or the following command
 
 ```bash
-az acr build --image <IMAGE_NAME>:<IMAGE_VERSION> --file Dockerfile --registry <ACR_REGISTRY> .
+python ddm_predictor.py --workspace-setup=True
```
Comment (akzaidi) [inline README.md]: Should be `python ddm_predictor.py simulator.workspace_setup=True` (which is usually the default)

**Review by akzaidi — APPROVED** [review]

**Comment by akzaidi** [comment]
Don't worry about the pytest failure (it's just for the torch_models which failed because of the torch removal from the requirements.txt).

File: README.md (hunk 1 above)
Comment (journeyman-msft) [reply README.md:49]: Either should work? Then I can remove this. This was one of the things Sandip suggested for when I finally got things to work with ddm_predictor.py

**Pull request merged**